### ISSUE TITLE
Request: tree-sitter dependency bump and new version release

### ISSUE THREAD

**Issue by WillLillis** [RubixDev/tree-sitter-asm#14]
Hi! Would it be possible to update the grammar's tree-sitter dependency to a newer version and publish a new release to crates.io? We've really enjoyed using the grammar in [asm-lsp](https://github.com/bergercookie/asm-lsp). However, I believe we're running into a bug with tree-sitter 0.22.10 and cannot upgrade until tree-sitter-asm has its dependency bumped as well. Testing locally at least, I think this is the only change that's necessary w.r.t. the rust bindings: 

```diff
diff --git a/Cargo.toml b/Cargo.toml
index 2494945..158a564 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,7 +26,7 @@ tree-sitter-c2rust = ["dep:tree-sitter-c2rust", "dep:tree-sitter-wasm-build-tool
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter-standard = { package = "tree-sitter", version = "0.20.9", optional = true }
+tree-sitter-standard = { package = "tree-sitter", version = "0.22.6", optional = true }
 tree-sitter-c2rust = { version = "0.20.9", optional = true }
 
 [build-dependencies]
```

**Comment by RubixDev** [comment]
I just published v0.22.6 (now following the tree-sitter version). Please try if this works for you, and if it does, feel free to close this issue.

**Comment by WillLillis** [comment]
Works for me, thank you!
